### ISSUE TITLE
Weblate+GitHub integration

### ISSUE THREAD

**Issue by UncombedCoconut** [naev/naev#1672]
There are a few things I can't do:

1. Set permissions for automatic push: Add the github username "Weblate" (display name "Weblate push user") to naev/naev as a collaborator with write permission. ([doc](https://docs.weblate.org/en/latest/vcs.html#vcs-repos))
2. Enable automatic push: (I can do this after step 1.) Go to [the settings page](https://hosted.weblate.org/settings/naev/naev/#vcs), add `git@github.com:naev/naev.git` as the " Repository push URL". We can decide when it tries to commit. I think a permissive/automatic model makes sense for now. (My attitude is: we probably can't have nice things in the long run, but why not make it easy until *after* we've had to revert someone's "every string in de.po is 'penis' now" commit.)
3. If not already done, add Weblate's hook to naev/naev's "Webooks" settings. ([doc](https://docs.weblate.org/en/latest/admin/continuous.html#github-setup))

**Comment by bobbens** [comment]
Done.
